### PR TITLE
Added infoseek.co.jp

### LINKED ISSUE #5825: Infoseek.co.jp
owned by rakuten.co.jp (#5824)

### PR DESCRIPTION
**Pull request by luixxiul**
Closes #5825

### CODE REVIEW

**Comment by J0WI** [comment]
Please have a look at our [guide](https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md) to find even more subdomains.

**Comment by luixxiul** [comment]
ok I'm checking

**Comment by luixxiul** [comment]
That is all I got from google

**Comment by luixxiul** [comment]
Updated, please review.

File: src/chrome/content/rules/infoseek.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+	Problematic subdomains:
+
+		- flash.api.earthquake	(invalid cert)
+		- mero	(invalid cert)
+
+-->
+<ruleset name="Infoseek" platform="mixedcontent">
+
+	<target host="image.infoseek.rakuten.co.jp" />
```
Comment (J0WI) [inline src/chrome/content/rules/infoseek.xml]: Is rakuten.co.jp related with infoseek.co.jp?
Please add a comment why this needs to be in this ruleset or put it in a another file.

File: src/chrome/content/rules/infoseek.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+			- transfer *
+			- translation *
+			- point ²
+			- schedule **
+			- special **
+
+	* Times out
+	² Redirects to http
+	** Refused
```
Comment (J0WI) [inline src/chrome/content/rules/infoseek.xml]: also for area.

File: src/chrome/content/rules/infoseek.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+-->
+<ruleset name="Infoseek" platform="mixedcontent">
+
+	<target host="image.infoseek.rakuten.co.jp" />
+	<target host="media.image.infoseek.co.jp" />
+	<target host="news.infoseek.co.jp" />
+	<target host="quiz.infoseek.co.jp" />
+	<target host="woman.infoseek.co.jp" />
+	<target host="www.infoseek.co.jp" />
```
Comment (J0WI) [inline src/chrome/content/rules/infoseek.xml]: You should also add ^infoseek.co.jp

**Comment by luixxiul** [comment]
@J0WI OK, updated.

File: src/chrome/content/rules/Infoseek.co.jp.xml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+<ruleset name="Infoseek" platform="mixedcontent">
+
+	<target host="infoseek.co.jp" />
+	<target host="media.image.infoseek.co.jp" />
+	<target host="news.infoseek.co.jp" />
+	<target host="quiz.infoseek.co.jp" />
+	<target host="woman.infoseek.co.jp" />
+	<target host="www.infoseek.co.jp" />
+
+	<test url="http://image.infoseek.rakuten.co.jp/content/news/manga/istop/loader.gif" />
```
Comment (J0WI) [inline src/chrome/content/rules/Infoseek.co.jp.xml]: This test url is now obsolete.

**Comment by luixxiul** [comment]
Updated.

**Pull request merged**